src/Microsoft.AspNetCore.SignalR/HubEndPoint.cs

```diff
@@ -63,9 +63,16 @@ public class HubEndPoint<THub, TClient> : EndPoint where THub : Hub<TClient>
 
                 using (var scope = _serviceScopeFactory.CreateScope())
                 {
-                    var hub = scope.ServiceProvider.GetService<THub>() ?? Activator.CreateInstance<THub>();
-                    InitializeHub(connection, hub);
+                    bool created;
```

Comment:
`bool created` is fine. In general I find this pattern ugly and it is now across the entire file but honestly, it works.